src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/changes-since-v3.mdx

```diff
@@ -7,6 +7,8 @@ tags:
 metaDescription: Changes introduced in Kubernetes Integration version 3
 ---
 
+import kubernetesV3Integration from 'images/kubernetes-v3-integration.svg'
```

Comment:
the build is failing because of this import statement, should be: 
```suggestion
import kubernetesV3Integration from 'images/kubernetes-v3-integration.png'
```
since the file uploaded is a `png`!!